Graphics/UI/Gtk/WebKit/WebView.chs

```diff
@@ -767,37 +774,35 @@ webViewGetPasteTargetList = maybePeek mkTargetList <=< {#call web_view_get_paste
 
 -- * Attibutes
 
+#if WEBKIT_CHECK_VERSION(1,0,1)
 -- | Zoom level of the 'WebView' instance
 webViewZoomLevel :: WebViewClass self => Attr self Float
-webViewZoomLevel = newAttr
-   webViewGetZoomLevel
-   webViewSetZoomLevel
+webViewZoomLevel = newAttrFromFloatProperty "zoom-level"
```

Comment:
Looking in the webkit code it looks like webkit_web_get/set_view_zoom_level (the C functions used by webViewGet/SetZoomLevel) are much more direct.  Are we sure we want to introduce the overhead of looking up the property by name?